.github/workflows/AI.yml

```diff
@@ -115,4 +115,4 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.G_TOKEN }}
           OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
           OPENAI_API_MODEL: "gpt-4o-mini"
-          exclude: "**/*.json, **/*.md"
+          exclude: "**/*.json,**/*.md"
```

Comment:
There should be a space after the comma in the exclude pattern for better readability.